tests/unittests/server/routes/test_sample.py

```
@@ -110,6 +112,7 @@ def fixture_sample(self, dataset):
             ]
         )
         sample["primitive_field"] = "initial_value"
+        sample["capture_time"] = datetime.datetime.now()
```

Ruff (0.14.11)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)